pkg/storagev2/migration/migration.go

```diff
@@ -33,22 +35,41 @@ func Migrate(s storage.Store, sm StepsMap) error {
 				return err
 			}
 		} else {
-			// there is no next version defiend
+			// there is no next version defined
 			// so we stop here
 			return nil
 		}
 	}
 }
 
+// ValidateVersions checks versions if they are in order n (where n min version value), n+1, n+2, n+3... (all values are increasing orders)
 func ValidateVersions(sm StepsMap) error {
-	// TODO
-	// all versions should be in order n (where n min version value), n+1, n+2, n+3... (all values are increasing orders)
-	// 5,6,7,8 ok
-	// 5,6,7,8,10 nok
+	keys := make([]int, len(sm))
+	i := 0
+	for k := range sm {
+		keys[i] = int(k)
+		i++
+	}
```

Comment:
Can `i` variable be part of `for`-loop?

Also, `versions` might be more indicative name then `keys`.

Reply:
`i` can't be part of for range loop as per loop syntax.

Reply:
right, this is range of maps